React/CoreModules/RCTExceptionsManager.h

```diff
@@ -37,7 +36,7 @@ NS_ASSUME_NONNULL_BEGIN
 - (void)reportFatalException:(nullable NSString *)message
                        stack:(nullable NSArray<NSDictionary *> *)stack
                  exceptionId:(double)exceptionId;
-- (void)reportEarlyJsException:(std::string)errorMap;
+- (void)reportEarlyJsException:(nullable NSString *)errorMap;
```

Comment:
@Kudo please revert this change. It is not related to the original issue, and it breaks our internal tests.
Other than that, this solution looks good!